geode-junit/src/main/java/org/apache/geode/codeAnalysis/AnalyzeSerializablesJUnitTestBase.java

```diff
@@ -433,7 +433,7 @@ private void loadClasses() throws IOException {
     String ideaBuildDirName = Paths.get(getModuleName(), "out", "production", "classes").toString();
     System.out.println("ideaBuildDirName is " + ideaBuildDirName);
     String ideaFQCNBuildDirName = Paths.get("out", "production",
-        "org.apache.geode." + getModuleName() + ".main").toString();
+        "geode." + getModuleName() + ".main").toString();
```

Comment:
I wish there was a way to do this without fishing for an intellij (or gradle) directory name.

Reply:
I've looked at how ArchUnit finds classes and it's also classpath & search based but is considerably more complex, hooking into analysis classes in gradle, idea and in one case a class available starting in jdk 9.